modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -7043,19 +7043,24 @@ type Flamingos2Dynamic {
   lyotWheel: Flamingos2LyotWheel!
 
   """
-  Flamingos 2 mask, if any.
+  Flamingos 2 FPU, if any.
   """
-  mask: Flamingos2Mask
+  fpu: Flamingos2FpuMask
 
   """
-  Readout mode, if specified.
+  Flamingos 2 decker.
   """
-  readoutMode: Flamingos2ReadoutMode
+  decker: Flamingos2Decker!
+
+  """
+  Flamingos 2 readout mode.
+  """
+  readoutMode: Flamingos2ReadoutMode!
 
   """
-  Reads, if specified.
+  Flamingos 2 reads.
   """
-  reads: Flamingos2Reads
+  reads: Flamingos2Reads!
```

Comment:
`readoutMode` and `reads` are now required.  They will be generated from the long slit config and sent back to the ODB from Observe as for everything else.